### PR TITLE
Author and game class

### PR DESCRIPTION
**Pull request by Dadadon**
- Created an Author class with an association with the Item class
- Created a game class that inherits from the item class
**Implement methods:**
> add_item method in the Author class
- Takes an instance of the Item class as an input
- Adds the input item to the collection of items
- Add self as a property of the item object (by using the correct setter from the item object)

> can_be_archived?() in the Game class
- Overrides the method from the parent class
- returns true if the parent's method returns true AND if last_played_at is older than 2 years otherwise, it should return false

### CODE REVIEW

File: author.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+require 'uuid'
+require_relative 'items'
+
+class Author
+  attr_accessor :first_name, :last_name, :id, :items
```
Comment (jfoyarzo) [inline author.rb]: The `id` property is listed as private on the UML diagram, so I think it shouldn't have a getter or setter
image

**Review by jfoyarzo — COMMENTED** [review]
Great job @Dadadon , there's only one small observation I made on the Author class, the rest of the code looks ready to merge :)

File: author.rb (hunk 1 above)
Comment (jfoyarzo) [reply author.rb]: Oh, and the same goes for `items`

Comment (MattGomb) [reply author.rb]: @Dadadon , I agree with Felipe, if you fix that, you can merge

**Review by MattGomb — COMMENTED** [review]
# Hey @Dadadon 👋🏻,

## Great job so far! 🌟 
congrats-gif
There are a couple of errors that need to be fixed, like the ones mentioned by @jfoyarzo 

Things that are great about your project:

- git-flow is followed ✔️ 
- code looks clean  ✔️ 
- PR is descriptive and detailed ✔️ 

### Status: Required changes ♻️ :

- [ ] Please fix the below part, according to the previous review comments!

**Pull request merged**